source/patterns/@aws-solutions-constructs/aws-kinesisstreams-lambda/test/__snapshots__/test.kinesisstreams-lambda.test.js.snap

```diff
@@ -249,6 +252,16 @@ Object {
                 ],
               },
             },
+            Object {
+              "Action": "kinesis:DescribeStream",
```

Comment:
I didn't add this specific permissions, it looks like it was just missing from the latest snapshot.